helm/amazon-ec2-metadata-mock/values.yaml

```diff
@@ -5,21 +5,23 @@ image:
   tag: "v1.5.0"
   pullPolicy: "IfNotPresent"
 
+replicaCount: 1
+
 # nameOverride overrides the name of the helm chart
 nameOverride: ""
 # fullnameOverride overrides the name of the application
 fullnameOverride: ""
 
-# Create node OS specific daemonset(s). (e.g. "linux", "windows", "linux windows")
+# Create node OS specific deployment(s). (e.g. "linux", "windows", "linux windows")
 targetNodeOs: "linux"
```

Comment:
the windows and linux vars don't appear to be in the readme

Reply:
win..what?